### ISSUE TITLE
Blurred Cover Background

### ISSUE THREAD

**Issue by Syilo555** [fast4x/RiMusic#2173]
### Description

Using the blurred cover background can reduce the visibility of the artist name and song title.The blur adjustment slider is great and does help improve the visibility but can you please add another adjustment slider to adjust the background brightness. I think the text visibility can be increased more when you can reduce the background brightness. It also good for people who don't like bright screen!
![Screenshot_20240603-222713_Gallery](https://github.com/fast4x/RiMusic/assets/167092738/bb119212-bfa5-4198-8113-5e028e7c0033)

### Suggested Solution

Yah! Add another adjustment slider to adjust the background brightness for the blurred cover background.

### Additional Context

Thankk youuu soooo muchhh ❤

**Comment by fast4x** [comment]
Yes, blurred background is partially implemented, actually there is a bug in brightness
